rules/naming/src/ValueObject/VariableAndCallAssign.php

```diff
@@ -49,7 +49,7 @@ final class VariableAndCallAssign
     public function __construct(
         Variable $variable,
         Expr $expr,
-        Assign $assign,
```

Comment:
I have a feeling that I have to use another value object  without `Assign` for foreach. What do you think @TomasVotruba ?

Reply:
That can be a way